articles/hosted-pages/login/index.md

```diff
@@ -81,14 +85,14 @@ In order to get started customizing the login page, you'll first want to choose
 
 ### 3. Customization
 
-You can customize the login page at will right from the editor. If you use Lock, you can alter its behavior and appearance with [configuration options](/libraries/lock/configuration). If you are building a custom UI, you can style the login page to your own specifications.
+You can customize the login page at will right from the editor. If you use Lock, you can alter its behavior and appearance with [configuration options](/libraries/lock/configuration). If you are building a custom UI, you can style the login page to your own specifications. Note that the Lock library can also be used for embedded login, so some language in the Lock documentation reflects that use case.
 
-Remember that the login page customizations are per **tenant** rather than per application, so all changes to the page's appearance and/or behavior will apply to **all** login actions. When necessary, you can provide different pages to different applications via a method discussed later in this document.
+Remember that the login page customizations are per **tenant** rather than per application, so all changes to the page's appearance and/or behavior will apply to **all** login actions on your tenant. When necessary, you can provide different pages to different applications via a method discussed later in this document.
```

Comment:
"Remember that login page customizations are per **tenant** rather than per application. All changes to the login page's appearance and behavior..."